client/my-sites/plans/current-plan/index.jsx

```diff
@@ -154,6 +155,47 @@ class CurrentPlan extends Component {
 		return <PaidPlanThankYou />;
 	}
 
+	renderMain() {
+		const { selectedSiteId, selectedSite, showJetpackChecklist, translate } = this.props;
+		const isLoading = this.isLoading();
+		const currentPlanSlug = selectedSite.plan.product_slug;
+		const planTitle = getPlan( currentPlanSlug ).getTitle();
+		const planFeaturesHeader = translate( '{{planName/}} plan features', {
+			components: { planName: <>{ planTitle }</> },
+		} );
+
+		return (
+			<>
+				<PurchasesListing />
+
+				{ showJetpackChecklist && (
+					<Fragment>
+						<QueryJetpackPlugins siteIds={ [ selectedSiteId ] } />
+						<JetpackChecklist />
+					</Fragment>
+				) }
+
+				<div
+					className={ classNames( 'current-plan__header-text current-plan__text', {
+						'is-placeholder': { isLoading },
+					} ) }
+				>
+					<h1 className="current-plan__header-heading">{ planFeaturesHeader }</h1>
+				</div>
+				<AsyncLoad
+					require="calypso/blocks/product-purchase-features-list"
+					placeholder={ null }
+					plan={ currentPlanSlug }
+					isPlaceholder={ isLoading }
+				/>
+			</>
+		);
+	}
+
+	renderEcommerceTrialPage() {
+		return <div className="ecommerce-trial">{ this.renderMain() }</div>;
```

Comment:
It may be worth updating this to use `ecommerce-trial-wrapper` for the class so it's clearer.
```suggestion
		return <div className="ecommerce-trial-wrapper">{ this.renderMain() }</div>;
```